src/components/MapSettings/TimeSelector.js

```diff
@@ -41,7 +41,7 @@ export default class TimeSelector extends React.Component {
     );
   }
 
-  static timeLinkedTooltip = <Tooltip>Timestamp matching is activated</Tooltip>;
+  static timeLinkedTooltip = <Tooltip id={"Time Link Button"}>Timestamp matching is activated</Tooltip>;
```

Comment:
HTML id attr [must not contain whitespace](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/id)
Also, for pure string props, you don't have to use { } and typically people don't.